src/Stripe.net/Services/Checkout/Sessions/SessionService.cs

```diff
@@ -35,14 +32,14 @@ public virtual Task<Session> CreateAsync(SessionCreateOptions options, RequestOp
             return this.CreateEntityAsync(options, requestOptions, cancellationToken);
         }
 
-        public virtual Session Get(string sessionId, SessionGetOptions options = null, RequestOptions requestOptions = null)
+        public virtual Session Get(string id, SessionGetOptions options = null, RequestOptions requestOptions = null)
```

Comment:
You likely have to undo this one change. We've discussed this in the past that it was a breaking change to do this even if we can't imagine anyone relying on the param name.

Reply:
forgot this was potentially breaking, revered these param name changes

Reply:
👍